apps/point_cloud_editor/include/pcl/apps/point_cloud_editor/selection.h

```diff
@@ -60,6 +60,15 @@ class Selection : public Statistics
         registerStats();
     }
 
+    /// @brief Copy constructor.
+    /// @param selection a const reference to a selection object whose
+    /// properties will be copied.
+    Selection (const Selection& selection)
```

Comment:
Would we be able to default this as well?

Reply:
Yeah, I fixed in https://github.com/PointCloudLibrary/pcl/pull/4695/commits/00e2f43d8b8494ea2cf954761fb8c95c94fb7b7c. This change is pretty concise than the old one!